src/db/clients/PgClient.ts

```diff
@@ -110,4 +111,84 @@ export default class PgClient implements ServerInterface {
     client.release();
     return result.rows;
   }
+
+  /**
+   * Check if a table has rows where column is exactly value
+   * @param table table to check
+   * @param column attribute to check
+   * @param value value to compare against
+   * @returns list of rows matching condition
+   */
+  public async checkExact(
+    table: string,
+    column: string,
+    value: string
+  ): Promise<unknown[]> {
+    const client = await this.pool.connect();
+    const result = await client.query(
+      `select * from ${table} where ${column} = '${value}'`
+    );
+    client.release();
+    return result.rows;
+  }
+
+  /**
+   * Check if a table has rows where column contains value
+   * @param table table to check
+   * @param column attribute to check
+   * @param value value to compare against
+   * @returns list of rows matching condition
+   */
+  public async checkContains(
+    table: string,
+    column: string,
+    value: string
+  ): Promise<unknown[]> {
+    const client = await this.pool.connect();
+    const result = await client.query(
+      `select * from ${table} where ${column} like '${value}'`
+    );
+    client.release();
+    return result.rows;
+  }
+
+  public async checkID(data: string, table: string): Promise<unknown[]> {
+    const client = await this.pool.connect();
+    const result = await client.query(
+      `SELECT *
+      FROM   ${table} target
+      WHERE  NOT EXISTS (SELECT 1
+                         FROM   ${data} inputTable
+                         WHERE  inputTable.ID = target.ID)`
+    );
+    client.release();
+    return result.rows;
+  }
+
+  public async checkNumber(
+    table: string,
+    column: string,
+    value: number,
+    comparison: RowNumberOperations
+  ): Promise<unknown[]> {
+    const client = await this.pool.connect();
+    const result = await client.query(
+      `select * from ${table} where ${column} ${comparison} ${value}`
+    );
+    client.release();
+    return result.rows;
+  }
+
+  public async checkBoolean(
+    table: string,
+    column: string,
+    value: RowBooleanOperations
+  ): Promise<unknown[]> {
+    const client = await this.pool.connect();
+    const result = await client.query(
+      `select * from ${table} where ${column} = ${value}`
+    );
+    client.release();
+    return result.rows;
+  }
```

Comment:
What is the difference between checkBoolean and checkExact? the code is the same

Reply:
You're right, I'll combine the two